nav2_voxel_grid/test/voxel_grid_tests.cpp

```diff
@@ -172,6 +172,16 @@ TEST(voxel_grid, clearVoxelLineInMap) {
   vg.markVoxelInMap(0, 0, 5, 0);
   vg.clearVoxelLineInMap(0, 0, 0, 0, 0, 9, nullptr, 16, 0);
   EXPECT_EQ(vg.getVoxel(0, 0, 5), nav2_voxel_grid::FREE);
+
+  // Testing for min range for raytrace clearing
+  vg.markVoxelInMap(0, 0, 5, 0);
```

Comment:
Nice simple test for this, it would be good to have one for the obstacle layer's 2D as well

Reply:
We want to make sure the project doesn't drop coverage % due to this addition